app/views/course/assessment/question/text_responses/_comprehension_solution_fields.html.slim

```diff
@@ -0,0 +1,18 @@
+= content_tag_for(:tr, f.object, class: 'nested-fields') do
+  td = f.input :solution_type,
+               collection: Course::Assessment::Question::TextResponseComprehensionSolution.solution_types.keys,
+               label_method: lambda { |key| t('.'+key) },
```

Comment:
Travis Test 4:
Incorrect test result. Keys are used albeit not explicitly typed out.
```$ if [ "$GROUP" = "misc" ]; then bundle exec i18n-tasks unused; fi
warning: parser/current is loading parser/ruby24, which recognizes
warning: 2.4.0-compliant syntax, but you are running 2.4.2.
warning: please see https://github.com/whitequark/parser#compatibility-with-ruby-mri.
Unused keys (2) | i18n-tasks v0.9.20
+--------+--------------------------------------------------------------------------------------------+-------------+
| Locale | Key                                                                                        | Value       |
+--------+--------------------------------------------------------------------------------------------+-------------+
|   en   | course.assessment.question.text_responses.comprehension_solution_fields.compre_keyword     | Keyword     |
|   en   | course.assessment.question.text_responses.comprehension_solution_fields.compre_lifted_word | Lifted Word |
+--------+--------------------------------------------------------------------------------------------+-------------+
The command "if [ "$GROUP" = "misc" ]; then bundle exec i18n-tasks unused; fi" exited with 1.
```

Reply:
Use string interpolation instead, `i18n-tasks` is able to detect keys generated dynamically using interpolation.

`t(".#{key}")`

Reply:
Thank you for the suggestion!